frontend/TvCv19.Frontend/Domain/Repositories/PhysicianRepository.cs

```diff
@@ -3,62 +3,69 @@
 using System.Linq;
 using System.Threading.Tasks;
 using Dapper;
+using Microsoft.Extensions.Configuration;
 using MySql.Data.MySqlClient;
 
 namespace TvCv19.Frontend.Domain.Repositories
 {
 
 
-    public class PhyscianRepository : BaseRepository, IPhysicianRepository
+    public class PhyscianRepository : IPhysicianRepository
     {
-
         public async Task<Physician> AddPhysicianAsync(Physician physician)
         {
-            var id = Guid.NewGuid().ToString().Replace("-", string.Empty);
-            var sql = @$"INSERT INTO medecc.caregiver
-                        (id, name, location, hierarchy, supervisor_id)
-                         VALUES ('{id}', @Name, @Location, @Hierarchy, @SupervisorId)";
+            using var context = new MedeccContext();
+
+            physician.Id = Guid.NewGuid().ToString().Replace("-", string.Empty);
```

Comment:
need to change Guid.NewGuid().ToString().Replace("-", string.Empty);  Guid.NewGuid().ToString("N");